mutt/signal.c

```diff
@@ -63,6 +64,40 @@ static sig_handler_t SegvHandler = mutt_sig_exit_handler;
 volatile sig_atomic_t SigInt;   ///< true after SIGINT is received
 volatile sig_atomic_t SigWinch; ///< true after SIGWINCH is received
 
+/**
+ * exit_print_int - AS-safe version of printf("%d", n)
+ * @param n Number to be printed
+ */
+static void exit_print_int(int n)
+{
+  if (n < 0)
+  {
+    write(1, "-", 1);
+    n = -n;
+  }
+
+  for (/*void*/; n > 9; n / 10)
```

Comment:
```suggestion
  for (; n > 9; n = n / 10)
```

We already have lots of `for (; ...`
`n / 10` didn't _do_ anything.

Reply:
D'oh!  Thanks!